modules/user/client-react/containers/ForgotPassword.jsx

```diff
@@ -1,17 +1,34 @@
 import React from 'react';
 import { graphql, compose } from 'react-apollo';
 
+import { translate } from '@module/i18n-client-react';
+import { FieldError } from '@module/validation-common-react';
+
 import ForgotPasswordView from '../components/ForgotPasswordView';
 
 import FORGOT_PASSWORD from '../graphql/ForgotPassword.graphql';
 
 class ForgotPassword extends React.Component {
+  state = {
+    sent: false
+  };
+
+  onSubmit = ({ forgotPassword, t }) => async values => {
```

Comment:
Every container component contains the same code snippet. Is there any way to take it out and omit repetition?

Reply:
fix